drv/psc-seq-server/src/main.rs

```diff
@@ -526,10 +777,29 @@ impl Psu {
             // decision is that the "NewlyInserted" settle time starts after the
             // contacts are _done_ scraping, not when they start.
             (_, Present::No, _) => {
+                ringbuf_entry!(Event::Removed {
```

Comment:
It hasn't changed in this PR, but I think the above conditional would be more clear as
```
(PsuState::Present(..), Present::No, _)
```
(because otherwise you have to read the first branch and also realize that the only other `PsuState` is `Present`)

Reply:
done!